src/baklava/OptionParams.ts

```diff
@@ -0,0 +1,6 @@
+export default class OptionParams {
```

Comment:
Especially on this @tomproberts. Basically, I want to be able to communicate from the Custom baklava node to the `TextAreaOption`, but the only way of doing this is bt passing arguments when adding the option. We then need this class to be able to set values from the Custom baklava node and to read them from the TextArea option. Does that look good? We might want to have it more general to fit more purposes.

Reply:
Baklava already has an `InputOption` which is very similar (just doesn't have label next to the box) so you can look at the way they deal with text input

Reply:
Hmm, I looked at it but it still doesn't offer a way of communicating with the Option right? What we want to do here is being able from the Custom baklava node to set a flag `hasError` to true in order to add the class `is-invalid` to the textarea html tag.

Reply:
So what you could do, is set the value of a `TextArea` to be an object, e.g. `{ hasError: boolean, text: string }`. Then whenever you parse the code (and there's an error) in the `nodeUpdated` method of `Custom.ts`, you can simply get the `TextArea` option by doing something like `const option = this.options.get('Inline Code')`, and then with that option you can set the value from the `Custom.ts` node. Something like `option.value = { hasError: true, text: code }` should work, no?

That would be the way I would do it. Personally the `hasError` field makes sense to be part of the option's value, maybe you would disagree though.

Reply:
Then obviously inside the `TextArea.vue` component you can just set the class `is-invalid` by normal JavaScript by checking the field `hasError` of its `value`.

Reply:
That sounds great, I've just implemented this. Feel free to check! 🙂